Failed to build on ubuntu 23.04

Installed dependencies:
```Dockerfile
FROM ubuntu:23.04 as build_env
RUN apt-get update -y && \
	apt-get install -y \
# 		bpfcc-tools \
		cargo \
		cmake \
		g++ \
		libgrpc++-dev \
		protobuf-compiler-grpc \
		rustc \
		systemtap-sdt-dev
RUN apt-get -y install arping bison git clang-format cmake dh-python \
  dpkg-dev pkg-kde-tools ethtool flex inetutils-ping iperf \
  libbpf-dev libclang-dev libclang-cpp-dev libedit-dev libelf-dev \
  libfl-dev libzip-dev linux-libc-dev llvm-dev libluajit-5.1-dev \
  luajit python3-netaddr python3-pyroute2 python3-setuptools python3
```
Build log:
```
git clone https://github.com/iovisor/bcc.git && 	mkdir bcc/build; cd bcc/build && 	cmake .. && 	make && 	make install
Cloning into 'bcc'...
-- The C compiler identification is GNU 12.3.0
-- The CXX compiler identification is GNU 12.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
Submodule 'libbpf-tools/blazesym' (https://github.com/libbpf/blazesym) registered for path 'libbpf-tools/blazesym'
Cloning into '/bcc/bcc/libbpf-tools/blazesym'...
Submodule path 'libbpf-tools/blazesym': checked out 'd954f73867527dc75025802160c759d0b6a0641f'
Submodule 'libbpf-tools/bpftool' (https://github.com/libbpf/bpftool) registered for path 'libbpf-tools/bpftool'
Submodule 'src/cc/libbpf' (https://github.com/libbpf/libbpf.git) registered for path 'src/cc/libbpf'
Cloning into '/bcc/bcc/libbpf-tools/bpftool'...
Cloning into '/bcc/bcc/src/cc/libbpf'...
Submodule path 'libbpf-tools/bpftool': checked out '6eb3e20583da834da18ea3011dcefd08b3493f8d'
Submodule 'libbpf' (https://github.com/libbpf/libbpf.git) registered for path 'libbpf-tools/bpftool/libbpf'
Cloning into '/bcc/bcc/libbpf-tools/bpftool/libbpf'...
Submodule path 'libbpf-tools/bpftool/libbpf': checked out '7984737fbf3b2a14a86321387bb62abb16cfc4ed'
Submodule path 'src/cc/libbpf': checked out '44b0bc9ad70a73cddc3528f12da48cc3520ad060'
-- Latest recognized Git tag is v0.28.0
-- Git HEAD is ec49363e2e9daec026ee6cae4c5fc316f8fab0ff
-- Revision is 0.28.0+ec49363e (major 0, minor 28, patch 0)
-- Performing Test HAVE_NO_PIE_FLAG
-- Performing Test HAVE_NO_PIE_FLAG - Success
-- Performing Test HAVE_REALLOCARRAY_SUPPORT
-- Performing Test HAVE_REALLOCARRAY_SUPPORT - Success
-- Kernel release: 6.3.13-linuxkit
-- Kernel headers: KERNELHEADERS_DIR-NOTFOUND
-- Performing Test HAVE_FFI_CALL
-- Performing Test HAVE_FFI_CALL - Success
-- Found FFI: /usr/lib/x86_64-linux-gnu/libffi.so
-- Performing Test Terminfo_LINKABLE
-- Performing Test Terminfo_LINKABLE - Success
-- Found Terminfo: /usr/lib/x86_64-linux-gnu/libtinfo.so
-- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.2.13")
-- Found LibXml2: /usr/lib/x86_64-linux-gnu/libxml2.so (found version "2.9.14")
-- Found LLVM: /usr/lib/llvm-15/include 15.0.7 (Use LLVM_ROOT envronment variable for another version of LLVM)
-- Found BISON: /usr/bin/bison (found version "3.8.2")
-- Found FLEX: /usr/bin/flex (found version "2.6.4")
-- Found LibElf: /usr/lib/x86_64-linux-gnu/libelf.so
-- Performing Test ELF_GETSHDRSTRNDX
-- Performing Test ELF_GETSHDRSTRNDX - Success
-- Could NOT find LibDebuginfod (missing: LIBDEBUGINFOD_LIBRARIES LIBDEBUGINFOD_INCLUDE_DIRS)
-- Could NOT find LibLzma (missing: LIBLZMA_LIBRARIES LIBLZMA_INCLUDE_DIRS)
-- Using static-libstdc++
-- Found LuaJIT: /usr/lib/x86_64-linux-gnu/libluajit-5.1.a;/usr/lib/x86_64-linux-gnu/libdl.a;/usr/lib/x86_64-linux-gnu/libm.so
CMake Warning at tests/python/CMakeLists.txt:10 (message):
  Recommended test program 'netperf' not found
-- Configuring done
-- Generating done
-- Build files have been written to: /bcc/bcc/build
[  1%] Building CXX object src/cc/frontends/clang/CMakeFiles/clang_frontend-objects.dir/loader.cc.o
[  1%] Building CXX object src/cc/frontends/clang/CMakeFiles/clang_frontend-objects.dir/b_frontend_action.cc.o
[  1%] Building CXX object src/cc/frontends/clang/CMakeFiles/clang_frontend-objects.dir/tp_frontend_action.cc.o
[  2%] Building CXX object src/cc/frontends/clang/CMakeFiles/clang_frontend-objects.dir/kbuild_helper.cc.o
/bcc/bcc/src/cc/frontends/clang/kbuild_helper.cc:84:3: warning: multi-line comment [-Wcomment]
   84 |   // USERINCLUDE    := \
      |   ^
/bcc/bcc/src/cc/frontends/clang/kbuild_helper.cc:93:3: warning: multi-line comment [-Wcomment]
   93 |   // LINUXINCLUDE    := \
      |   ^
[  2%] Built target clang_frontend-objects
[  2%] Linking CXX static library libclang_frontend.a
[  2%] Built target clang_frontend
[  3%] Building C object src/cc/CMakeFiles/bpf-static.dir/libbpf.c.o
[  3%] Building C object src/cc/CMakeFiles/bpf-static.dir/perf_reader.c.o
[  3%] Building C object src/cc/CMakeFiles/bpf-static.dir/libbpf/src/bpf.c.o
[  4%] Building C object src/cc/CMakeFiles/bpf-static.dir/libbpf/src/bpf_prog_linfo.c.o
[  4%] Building C object src/cc/CMakeFiles/bpf-static.dir/libbpf/src/btf.c.o
[  4%] Building C object src/cc/CMakeFiles/bpf-static.dir/libbpf/src/btf_dump.c.o
[  5%] Building C object src/cc/CMakeFiles/bpf-static.dir/libbpf/src/gen_loader.c.o
[  5%] Building C object src/cc/CMakeFiles/bpf-static.dir/libbpf/src/hashmap.c.o
[  5%] Building C object src/cc/CMakeFiles/bpf-static.dir/libbpf/src/libbpf.c.o
[  6%] Building C object src/cc/CMakeFiles/bpf-static.dir/libbpf/src/libbpf_errno.c.o
[  6%] Building C object src/cc/CMakeFiles/bpf-static.dir/libbpf/src/libbpf_probes.c.o
[  6%] Building C object src/cc/CMakeFiles/bpf-static.dir/libbpf/src/linker.c.o
[  6%] Building C object src/cc/CMakeFiles/bpf-static.dir/libbpf/src/netlink.c.o
[  7%] Building C object src/cc/CMakeFiles/bpf-static.dir/libbpf/src/nlattr.c.o
[  7%] Building C object src/cc/CMakeFiles/bpf-static.dir/libbpf/src/relo_core.c.o
[  7%] Building C object src/cc/CMakeFiles/bpf-static.dir/libbpf/src/ringbuf.c.o
[  8%] Building C object src/cc/CMakeFiles/bpf-static.dir/libbpf/src/str_error.c.o
[  8%] Building C object src/cc/CMakeFiles/bpf-static.dir/libbpf/src/strset.c.o
[  8%] Building C object src/cc/CMakeFiles/bpf-static.dir/libbpf/src/usdt.c.o
[  9%] Building C object src/cc/CMakeFiles/bpf-static.dir/libbpf/src/zip.c.o
[  9%] Building CXX object src/cc/CMakeFiles/bpf-static.dir/bcc_syms.cc.o
[  9%] Building C object src/cc/CMakeFiles/bpf-static.dir/bcc_elf.c.o
[ 10%] Building C object src/cc/CMakeFiles/bpf-static.dir/bcc_perf_map.c.o
[ 10%] Building C object src/cc/CMakeFiles/bpf-static.dir/bcc_proc.c.o
[ 10%] Building C object src/cc/CMakeFiles/bpf-static.dir/bcc_zip.c.o
[ 10%] Building CXX object src/cc/CMakeFiles/bpf-static.dir/common.cc.o
[ 11%] Building CXX object src/cc/CMakeFiles/bpf-static.dir/usdt/usdt.cc.o
[ 11%] Building CXX object src/cc/CMakeFiles/bpf-static.dir/usdt/usdt_args.cc.o
[ 11%] Linking CXX static library libbcc_bpf.a
[ 11%] Built target bpf-static
[ 12%] Building CXX object src/cc/api/CMakeFiles/api-static.dir/BPF.cc.o
[ 12%] Building CXX object src/cc/api/CMakeFiles/api-static.dir/BPFTable.cc.o
[ 12%] Linking CXX static library libapi-static.a
[ 12%] Built target api-static
[ 13%] Building CXX object src/cc/usdt/CMakeFiles/usdt-static.dir/usdt_args.cc.o
[ 13%] Building CXX object src/cc/usdt/CMakeFiles/usdt-static.dir/usdt.cc.o
[ 13%] Linking CXX static library libusdt-static.a
[ 13%] Built target usdt-static
[ 13%] Building CXX object src/cc/CMakeFiles/bcc-shared.dir/link_all.cc.o
[ 13%] Building CXX object src/cc/CMakeFiles/bcc-shared.dir/bcc_common.cc.o
[ 14%] Building CXX object src/cc/CMakeFiles/bcc-shared.dir/bpf_module.cc.o
[ 14%] Building CXX object src/cc/CMakeFiles/bcc-shared.dir/bcc_btf.cc.o
[ 14%] Building CXX object src/cc/CMakeFiles/bcc-shared.dir/exported_files.cc.o
[ 15%] Building CXX object src/cc/CMakeFiles/bcc-shared.dir/bcc_debug.cc.o
[ 15%] Building CXX object src/cc/CMakeFiles/bcc-shared.dir/bpf_module_rw_engine.cc.o
[ 15%] Building CXX object src/cc/CMakeFiles/bcc-shared.dir/table_storage.cc.o
[ 15%] Building CXX object src/cc/CMakeFiles/bcc-shared.dir/shared_table.cc.o
[ 16%] Building CXX object src/cc/CMakeFiles/bcc-shared.dir/bpffs_table.cc.o
[ 16%] Building CXX object src/cc/CMakeFiles/bcc-shared.dir/json_map_decl_visitor.cc.o
[ 16%] Building CXX object src/cc/CMakeFiles/bcc-shared.dir/bcc_syms.cc.o
[ 17%] Building C object src/cc/CMakeFiles/bcc-shared.dir/bcc_elf.c.o
[ 17%] Building C object src/cc/CMakeFiles/bcc-shared.dir/bcc_perf_map.c.o
[ 17%] Building C object src/cc/CMakeFiles/bcc-shared.dir/bcc_proc.c.o
[ 18%] Building C object src/cc/CMakeFiles/bcc-shared.dir/bcc_zip.c.o
[ 18%] Building CXX object src/cc/CMakeFiles/bcc-shared.dir/common.cc.o
make[2]: *** No rule to make target '/usr/lib/llvm-15/lib/libPolly.a', needed by 'src/cc/libbcc.so.0.28.0'.  Stop.
make[1]: *** [CMakeFiles/Makefile2:753: src/cc/CMakeFiles/bcc-shared.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
ERROR: process "/bin/sh -c git clone https://github.com/iovisor/bcc.git && \tmkdir bcc/build; cd bcc/build && \tcmake .. && \tmake && \tmake install" did not complete successfully: exit code: 2
```

@chantra thank you, libpolly helps 

```Dockerfile
FROM ubuntu:23.04 as build_env
RUN apt-get update -y && \
	apt-get install -y \
# 		bpfcc-tools \
		cargo \
		cmake \
		g++ \
		libgrpc++-dev \
		protobuf-compiler-grpc \
		rustc \
		systemtap-sdt-dev
RUN apt-get -y install arping bison git cmake dh-python \
	dpkg-dev pkg-kde-tools ethtool flex inetutils-ping iperf \
	libbpf-dev libedit-dev libelf-dev \
	libfl-dev libzip-dev linux-libc-dev libluajit-5.1-dev \
	luajit python3-netaddr python3-pyroute2 python3-setuptools python3
RUN apt-get install -y clang-16 \
	clang-format-16 \
	libclang-16-dev \
	libclang-common-16-dev \
	libclang-cpp16-dev \
	libclang1-16 \
	llvm-16 \
	llvm-16-dev \
	llvm-16-runtime \
	libllvm16 \
	libpolly-16-dev
WORKDIR /bcc
RUN git clone --recursive --branch=v0.28.0 https://github.com/iovisor/bcc.git && \
	mkdir bcc/build; cd bcc/build && \
	cmake .. && \
	make && \
	make install
```

zip also requried:
```
#12 895.9 [ 91%] Building CXX object tests/cc/CMakeFiles/usdt_test_lib.dir/usdt_test_lib.cc.o
#12 896.0 [ 91%] Linking CXX shared library libusdt_test_lib.so
#12 896.1 [ 91%] Built target usdt_test_lib
#12 896.2 [ 91%] Building CXX object tests/cc/CMakeFiles/debuginfo_test_lib.dir/debuginfo_test_lib.cc.o
#12 896.2 [ 92%] Linking CXX shared library libdebuginfo_test_lib.so
#12 896.3 [ 92%] Built target debuginfo_test_lib
#12 896.4 [ 93%] Generating archive.zip
#12 896.4 /bin/sh: 1: zip: not found
#12 896.4 make[2]: *** [tests/cc/CMakeFiles/zip_archive.dir/build.make:75: tests/cc/archive.zip] Error 127
#12 896.4 make[1]: *** [CMakeFiles/Makefile2:1704: tests/cc/CMakeFiles/zip_archive.dir/all] Error 2
#12 896.4 make: *** [Makefile:146: all] Error 2
#12 ERROR: process "/bin/sh -c git clone --recursive --branch=v0.28.0 https://github.com/iovisor/bcc.git && \tmkdir bcc/build; cd bcc/build && \t cmake .. && \tmake && \tmake install" did not complete successfully: exit code: 2
------
 > [app_env  6/19] RUN git clone --recursive --branch=v0.28.0 https://github.com/iovisor/bcc.git && 	mkdir bcc/build; cd bcc/build && 	 cmake .. && 	make && 	make install:
896.0 [ 91%] Linking CXX shared library libusdt_test_lib.so
896.1 [ 91%] Built target usdt_test_lib
896.2 [ 91%] Building CXX object tests/cc/CMakeFiles/debuginfo_test_lib.dir/debuginfo_test_lib.cc.o
896.2 [ 92%] Linking CXX shared library libdebuginfo_test_lib.so
896.3 [ 92%] Built target debuginfo_test_lib
896.4 [ 93%] Generating archive.zip
896.4 /bin/sh: 1: zip: not found
```

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -399,6 +399,11 @@ sudo apt install -y zip bison build-essential cmake flex git libedit-dev \
 sudo apt install -y zip bison build-essential cmake flex git libedit-dev \
   libllvm14 llvm-14-dev libclang-14-dev python3 zlib1g-dev libelf-dev libfl-dev python3-setuptools \
   liblzma-dev libdebuginfod-dev arping netperf iperf
+  
+# For Lunar Lobster (23.04)
+sudo apt install -y zip bison build-essential cmake flex git libedit-dev \
+  libllvm15 llvm-15-dev libclang-15-dev python3 zlib1g-dev libelf-dev libfl-dev python3-setuptools \
+  liblzma-dev libdebuginfod-dev arping netperf iperf libpolly-15-dev
 
 # For other versions
 sudo apt-get -y install zip bison build-essential cmake flex git libedit-dev \

